landing-pages/site/assets/scss/_list-boxes.scss

```diff
@@ -63,9 +63,11 @@ $card-margin: 20px;
   border-radius: 5px;
   margin: $card-margin;
   padding: 30px 10px;
+  height: calc(100% - #{$card-margin} * 2);
```

Comment:
Can you do it without calc? For example:: `align-items: stretch`

Reply:
resolved